ledger/process/cache_processor.go

```diff
@@ -206,7 +207,12 @@ func (lv *LedgerVerifier) BlockCacheProcess(block *types.StateBlock) error {
 	if err := lv.updateAccountMetaCache(block, am, batch); err != nil {
 		return fmt.Errorf("update account meta cache error: %s", err)
 	}
-	return lv.l.DBStore().PutBatch(batch)
+	err = lv.l.DBStore().PutBatch(batch)
+	if err != nil {
+		return err
+	}
+	lv.l.EB.Publish(topic.EventBroadcast, &topic.EventBroadcastMsg{Type: topic.PublishReq, Message: block})
```

Comment:
```suggestion
	lv.l.EB.Publish(topic.EventBroadcast, &topic.EventBroadcastMsg{Type: topic.PublishReq, Message: block})
	lv.l.EB.Publish(topic.EventGenerateBlock, block)
```

Reply:
before is in LedgerAPI.Process,now is in verify.BlockCacheProcess